ci/region.sh

```diff
@@ -43,6 +43,10 @@ case "$1" in
     echo AWS_REGION=ap-southeast-1
     echo RACK_PRIVATE=yes
     ;;
+  9)
+    echo AWS_DEFAULT_REGION=eu-west-2
```

Comment:
Could we group the regions together here. That is make `eu-west-2` number 5 and move the others down. `ap-southeast-2` is floating up there too.